RedacteurPortaal.Api/Controllers/AddressController.cs

```diff
@@ -15,35 +15,30 @@ namespace RedacteurPortaal.Api.Controllers
     public class AddressController : Controller
     {
         private readonly IGrainManagementService<IAddressGrain> grainService;
-        private readonly ILogger logger;
-        
-        public AddressController(IGrainManagementService<IAddressGrain> grainService, ILogger<AddressController> logger)
+
+        public AddressController(IGrainManagementService<IAddressGrain> grainService)
         {
             this.grainService = grainService;    
-            this.logger = logger;
         }
 
         [HttpPost]
         public async Task<ActionResult<AddressDTO>> SaveAddress([FromBody] AddAddressRequest addressDTO )   
         {
             Guid newguid = Guid.NewGuid();
-            TypeAdapterConfig<AddressDTO, AddressModel>
-                .NewConfig()
-                .Map(dest => dest.Id,
-                    src => newguid);
-            
+
             var address = addressDTO.Adapt<AddressModel>();
-            var grain = await this.grainService.CreateGrain(address.Id);
+            var grain = await this.grainService.CreateGrain(newguid);
             var updatedGrain = await grain.UpdateAddress(address);
             var response = updatedGrain.Adapt<AddressDTO>();
-            return this.CreatedAtRoute("GetAddress", new { id = newguid }, response);
+            return this.CreatedAtRoute("GetAddress", new { id = newguid.ToString() }, response);
         }
 
         [HttpGet]
         [Route("{id}", Name = "GetAddress")]
-        public async Task<AddressDTO> GetAddress(Guid id)
+        public async Task<AddressDTO> GetAddress(string id)
```

Comment:
Wat is hier de reden dat je de id als string opvraagt?

Reply:
Was nog overgebleven van een uitprobeersel, heb t verwijdert.